package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "shoping-list",
+  "version": "1.0.0",
+  "description": "shoping-list",
+  "main": "app.js",
+  "scripts": {
+    "test": "jest ./tests/test.js",
```

Comment:
why just `test.js`? I'd say you can put something like `"NODE_ENV=test jest --verbose --maxWorkers=5"`

Reply:
and it looks like there's some unresolved promise hanging, jest waits for a long time after the last test passes before it quits

Reply:
is unresolved promise issue in code or in test? Why is it hanging if tests have been run?

Reply:
Both review comments solved by ""test": "jest ./tests/ NODE_ENV=test jest --verbose --maxWorkers=5 --forceExit""